doc/source/install.rst

```diff
@@ -1,3 +1,17 @@
+Requirements
+============
+
+GreenplumPython currently requires at least Python 3.9 to run, this is because:
+    * Python 3.9 is the version we officially support and release with PL/Python3 and GPDB 6.
+    * Python 3.9 is the default version in Rocky Linux 9 and is officially supported in Rocky Linux 8 (and also probably in RHEL 8 as well).
+
+As a dependency of GreenplumPython, psycopg2 is a source package. To install it, you need the following on the client side
```

Comment:
Do we still need to mention this since we changed to use `psycopg2-binary` in the `requirements.txt`?

Reply:
OK, @yihong0618 let's remove this part to keep a short and efficient document